### DISCUSSION TITLE
Undo warning as error `-Wno-error=incompatible-function-pointer-types` fails

### DISCUSSION THREAD

**Discussion by vadimkantorov** [emscripten-core/emscripten#23033] [General]
`emcc -c   -D_GNU_SOURCE -D_POSIX_C_SOURCE -DSTANDARD_C -DPERL_USE_SAFE_PUTENV -DNO_MATHOMS -Wno-null-pointer-arithmetic -Wno-error=incompatible-function-pointer-type -pipe -fstack-protector-strong -Wall -O2   -DVERSION=\"2.20\" -DXS_VERSION=\"2.20\"  "-I../.."   POSIX.c
warning: unknown warning option '-Werror=incompatible-function-pointer-type'; did you mean '-Werror=incompatible-function-pointer-types'? [-Wunknown-warning-option]`

I'm trying to override the following error via passing `-Wno-error=incompatible-function-pointer-types`. How to do this with emcc/clang?
```
POSIX.xs:2909:48: warning: comparison of distinct pointer types ('void (*)(int, siginfo_t *, void *)' and 'Sighandler3_t' (aka 'void (*)(int, Siginfo_t *, void *)')) [-Wcompare-distinct-pointer-types]
 2909 |                         ? (  oact.sa_sigaction == PL_csighandler3p
      |                              ~~~~~~~~~~~~~~~~~ ^  ~~~~~~~~~~~~~~~~
POSIX.xs:2941:38: error: incompatible function pointer types assigning to 'void (*)(int, siginfo_t *, void *)' from 'Sighandler3_t' (aka 'void (*)(int, Siginfo_t *, void *)') [-Wincompatible-function-pointer-types]
 2941 |                     act.sa_sigaction =
      |                                      ^
 2942 |                             safe ? PL_csighandler3p : PL_sighandler3p;
      |                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

**Comment by sbc100** [comment]
Looks like you are missing the `s` on the end of `-Wno-error=incompatible-function-pointer-type`

**Reply by vadimkantorov** [reply]
ouch :)
